rskj-core/src/main/java/org/ethereum/rpc/exception/RskJsonRpcRequestException.java

```diff
@@ -1,37 +1,65 @@
 package org.ethereum.rpc.exception;
 
+import co.rsk.rpc.modules.eth.ProgramRevert;
+
+import javax.annotation.Nullable;
+
 public class RskJsonRpcRequestException extends RuntimeException {
 
     private final Integer code;
 
-    protected RskJsonRpcRequestException(Integer code, String message, Exception e) {
+    private final byte[] revertData;
```

Comment:
In JSON-RPC 2.0 specification there is a field `data` into the error response [(link) ](https://www.jsonrpc.org/specification#error_object) so we can update this attribute name from `revertData` to only `data` and then could be used by other methods if needed.

Reply:
sure but this is not an RPC response yet, so I'd say that overspecifying is better generalising for clarity sake. wdyt?